src/test/d8_known_gcc_test_failures.txt

```diff
@@ -100,16 +100,3 @@ va-arg-22.c.s.wast.wasm # abort()
 # LLVM emits a divide by zero:
 #   https://llvm.org/bugs/show_bug.cgi?id=26452
 pr60960.c.s.wast.wasm # divide by zero
-
-# New in 0xc:
-# abort() gets called
-930126-1.c.s.wast.wasm
```

Comment:
@jgravelle-google This one should pass after you land https://github.com/WebAssembly/binaryen/pull/731